src/app/embed/embed.component.ts

```diff
@@ -42,6 +45,8 @@ export class EmbedComponent implements OnInit {
   showPlaylist: boolean;
   episodes: AdapterProperties[];
 
+  @ViewChild(PlayerComponent) private player: PlayerComponent;
```

Comment:
What's this for?  Just not using this ref yet?

Reply:
Is the idea that you'd call `player.displayOverlay()` somewhere in this component?  Would it be easier to pass some sort of `@Input() showOverlay = false` into the player component?

Reply:
yes, this all got covered in the next PR